pkg/codegen/docs/gen.go

```diff
@@ -1514,6 +1539,23 @@ func (mod *modContext) genIndex() indexData {
 	return data
 }
 
+func formatTitleText(title string) string {
+	// If title not found in titleLookup map, default back to title given.
+	if val, ok := titleLookup[title]; ok {
+		return val
+	}
+	return title
+}
+
+func decodeLangSpecificInfo(pkg *schema.Package, lang string, obj interface{}) error {
```

Comment:
I think you may have made a slight error in resolving the conflict here. This function was actually removed with Pat's recent PR as it is no longer needed. See https://github.com/pulumi/pulumi/pull/4454/files#diff-ffd570ac00ee5f55462913956726d9d9L1518.